### PR TITLE
Problems2 D-Bus API

### PR DESCRIPTION
**Pull request by jfilak**

### CODE REVIEW

**Comment by jfilak** [comment]
I have to change the tests a bit as our testing machines cannot create 2GiB files.

File: src/lib/notify_new_path.c [hunk 1]
```diff
@@ -34,18 +42,68 @@ void notify_new_path(const char *path)
...
      * aka "good practice".
      */
     shutdown(fd, SHUT_WR);
+    if (message == NULL)
+    {
+        close(fd);
+        return 0;
+    }
+
+    *message = NULL;
```
Comment (kdudka) [inline src/lib/notify_new_path.c]: Looks like a redundant statement to me.

**Comment by jfilak** [comment]
I reduced disk space consumption to hundreds of megabytes from gigabytes

File: src/lib/notify_new_path.c (hunk 1 above)
Comment (jfilak) [inline src/lib/notify_new_path.c]: Fixed, thanks!

**Comment by mhabrnal** [comment]
Otherwise looks good to me, test works as well.

**Comment by jfilak** [comment]
I pushed updated commits to master. Tests passes, `make rpm` works.

**Pull request closed without merging**